hnn_core/gui/gui.py

```diff
@@ -228,6 +227,12 @@ def __init__(self, theme_color="#8A2BE2",
                                             max=multiprocessing.cpu_count(),
                                             description='Cores:',
                                             disabled=False)
+        self.load_data_button = FileUpload(
+            accept='.txt', multiple=False,
+            style={'button_color': self.layout['theme_color']},
+            description='Load data',
+            # layout=self.layout['btn_full_w'],
```

Comment:
delete before merge ...

Reply:
Sure, but before that, do you have any ideas on where to place this button, or does the current location look fine to you guys?

Reply:
I was thinking next to the "Run" button?